cmd/nem/main.go

```diff
@@ -200,6 +219,39 @@ func vanityAction(c *cli.Context) error {
 	return nil
 }
 
+func info(c *cli.Context) error {
+	ch, err := chainGlobalOption(c)
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	reader := bufio.NewReader(os.Stdin)
+	privateKeyStr, err := reader.ReadString('\n')
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	pkBytes, err := hex.DecodeString(strings.TrimSpace(privateKeyStr))
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	pair := keypair.Gen(bytes.NewReader(pkBytes))
+	if c.Bool("address-only") {
+		printAddress(ch, pair)
+	}
+
+	if c.Bool("public-only") {
+		printPublicKey(pair)
+	}
+
+	if !c.Bool("address-only") && !c.Bool("public-only") {
```

Comment:
As `else` can be better, or?

Reply:
Than user can't specify 2 parameters simultaniously -a,  -p